error-monitoring/src/bot.ts

```diff
@@ -59,6 +60,16 @@ export class ErrorIssueBot {
     };
   }
 
+  /** Comments on an existing issue to link a duplicate error. */
+  async commentWithDupe(errorId: string, issueNumber: number) {
+    await this.octokit.issues.createComment({
+      owner: this.repoOwner,
+      repo: this.issueRepoName,
+      'issue_number': issueNumber,
```

Comment:
Ugh I hate this... Prettier ain't pretty >__<

Reply:
Agreed >__>